pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -665,11 +665,7 @@ func (b *volumeBinder) findMatchingVolumes(pod *v1.Pod, claimsToBind []*v1.Persi
 
 	for _, pvc := range claimsToBind {
 		// Get storage class name from each PVC
-		storageClassName := ""
-		storageClass := pvc.Spec.StorageClassName
-		if storageClass != nil {
-			storageClassName = *storageClass
-		}
+		storageClassName := v1helper.GetPersistentVolumeClaimClass(pvc)
```

Comment:
I think https://github.com/kubernetes/kubernetes/blob/c5635753ea5b148b6f416d5692beeb221f29a76e/pkg/scheduler/algorithm/predicates/csi_volume_predicate.go also has to be updated

Reply:
Thank you for your review. I fixed it.